src/web/make-graph.rkt

```diff
@@ -56,12 +56,12 @@
    [_ #f]))
 
 (define (make-graph result out fpcore? profile?)
-  (match-define (job-result test _ time _ warnings backend) result)
+  (match-define (job-result test _ time data-list warnings backend) result)
+  (define bogosity (hash-ref (second data-list) 'bogosity))
```

Comment:
No, please do *not* extract the bogosity data from the timeline. The timeline should only go to the timeline page. Please plumb it through as a field on the `job-result` `backend` field.